python/pycylon/data/csv.pyx

```diff
@@ -47,4 +48,17 @@ def read_csv(context, path, csv_read_options) -> Table:
         if status.is_ok():
             return pycylon_wrap_table(cn_table)
         else:
-            raise Exception(f"Table couldn't be created from CSV: {status.get_msg().decode()}")
\ No newline at end of file
+            raise Exception(f"Table couldn't be created from CSV: {status.get_msg().decode()}")
+
+def to_csv(table, path, csv_write_options):
+        """
+        creating a csv file with PyCylon table data
```

Comment:
I'm not sure in Python API, we should associate these with the table?

Reply:
@vibhatha could you comment

Reply:
@supunkamburugamuve  I am taking a look.

Reply:
as I can remember I made this to_csv as a function in the table. 
for csv.pyx we shouldn't have dependencies with the table class. 
correct me if I am wrong.

Reply:
Yes, to_csv was originally a part of Table. But with WriteCSV being moved out of Table, in the cpp code, I mirrored the same with the Python code (and altered any instances where it was used like in tests). 
Should the Python code be kept as it was? Where to_csv is a part of Table in Python but not in the cpp version?

Reply:
The reasoning behind was to keep the Python API as user friendly as possible. Lets keep it as previously for now.

Reply:
Will do. I'll make the changes and update the pull request.

Reply:
+1